src/index.mjs

```diff
@@ -156,7 +156,7 @@ export function listen(options = {}) {
             });
           }
         }, delay);
-      // On exit
+        // On exit
```

Comment:
There are still some unrelated changes, please revert these

Reply:
I just pushed the fix. Thanks